.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
 ## Description
 Describe your pull request.
 
-## Associated Issue
-Issue #???
+## Associated / Resolved Issue
+Resolves # or link to clickup card
```

Comment:
```suggestion
Resolves Issue # or [ClickUp card](link-to-clickup-card)
```

Reply:
If you put an issue issue number after the # it automatically resolved it for you. See associated issue for PR #1909

Reply:
Gotcha